### ISSUE TITLE
Error on initialization

### ISSUE THREAD

**Issue by jronimo** [Simonwep/selection#30]
Very cool plugin. Thanks for creating.

I downloaded the plugin from GitHub today.

I'm using Chrome Version 73.0.3683.86 (Official Build) (64-bit).

My operating system is Windows 10.

I'm having a go at getting the plugin running for the first time.

I am getting an exception on drag start. 

My bare-bones instantiation code is,

					const selection = Selection.create({
						
						class: 'viewport',
						selectables: ['.cell'],
						startareas: ['html'],
						boundaries: ['html'],
						selectionAreaContainer: 'body',
					});

I have a div with the class 'viewport'. It contains child div elements with the class 'cell'.

I've attached a screenshot of the error message. In case there is a problem with the attachment, the text of the error message is, "selection.min.js:7 Uncaught TypeError: t._targetContainer.getBoundingClientRect is not a function at HTMLDocument._onTapStart (selection.min.js:7)".

![Screenshot](https://user-images.githubusercontent.com/6685860/55663118-f52bc500-585d-11e9-9a86-fd9f12334bf8.jpg)

**Comment by Simonwep** [comment]
Ugh, that's strange... First I would suggest another class-name for your selection-area since you already used `.viewport` as container for your divs. e.g: 
```javascript
class: 'my-selection-area',
selectables: ['.cell'],
startareas: ['html'],
boundaries: ['html'],
selectionAreaContainer: 'body'
```

Please keep in mind to add some fancy css to your `.my-selection-area` so the selection-area is visible:
```css
.my-selection-area {
    background: rgba(0, 0, 255, 0.1);
    border: 0.05em solid rgba(0, 0, 255, 0.2);
    border-radius: 0.1em;
}
```

If this issue persists, could you create a [JSFiddle](https://jsfiddle.net/)?
It's also require to implement the selection-handling-related stuff (since it should provide maximum customization possibilities), see the [demo](https://simonwep.github.io/selection/) for further instructions.

**Comment by Simonwep** [comment]
Okay, managed to reproduce it. There are some issue if you use `html` and / or other elements are overflowing it. Try to use at least the `body` element as `startareas` and `boundaries`. The best would be to restrict the entire selection stuff to a div :)

I've made a small bug fix where this problem get's visible: 168dadc1d1ee91f403e3172100608fc77413661f Try it with the lastest master version!
